rules/how-to-monetize-apps/rule.md

```diff
@@ -14,66 +16,87 @@ redirects:
+You don't have to charge users for your software. In fact, even free apps and games are designed to make money. Do you know the best way to monetize your apps?
 
 <!--endintro-->
 
+Apps make money in all sorts of ways, even the 'free' ones. You need to choose the right monetization model for your apps depending on it's functionality and audience.
 
+## Free vs Paid Apps
+Some apps require a form of payment, meaning you must pay for the app before you can use it. Others are 'free', meaning you can use the software, and all it's features, without paying for it. These apps make money in other ways. The middle ground is called 'freemium', which is where you get the basic app or game for free, but need to pay to access advanced features.
 
+## Payment Models
+Whether your app is paid up front or freemium, if you need to collect money at some point from your users, there are a few ways to do it:
 
-::: bad
-Bad example: paid apps
-
-:::
-
+### One-time fee
+This is the traditional approach, where you set a price for your app or game, and users must buy it before they can use it. This might be the right approach for you if:
+* Your product is a AAA gaming title
+* Your product is a useful, but not essential, productivity tool
+* Your product is *NOT* a SaaS product (one-off payments are difficult to reconcile against ongoing cloud and hosting costs)
 
 ::: good
-Good example: free apps with in-app purchases  
-:::
-
-
-
-
-::: bad
-Bad example: paid with currency  
+![Figure: The Legend of Engadine on iOS and Android is free to play, but a one-time purchase unlocks premium features.](engadine.png)
+**Good Example:** [The Legend of Engadine](https://engadine.net) on iOS and Android is free to play, but a one-time purchase unlocks premium features.
 :::
+    
+One-time fees work with either paid apps or freemium apps; users can either buy the app up-front or use the app for free, and pay to unlock premium features.
 
+### Subscription fee
+Subscriptions have become much more popular, even for traditional desktop software. Rather than pay a one-time fee, users pay a recurring monthly or annual fee to use the software. This might be the right approach for you if:
+* Your product is a SaaS product
+* Your product is likely to get recurring regular use *without* you needing to continually drive engagement (e.g., an essential, or at least high value, productivity tool)
+* Your product has other ongoing maintenance costs (e.g. content development)
 
 ::: good
-Good example: paid with abstract currency  
+![Figure: Microsoft 365 is available to home and business users for a low recurring cost.](microsoft365.png)
+**Good Example:** [Microsoft 365](https://www.microsoft.com/microsoft-365) (formerly Office 365) is a suite of productivity tools that run on mobile, desktop and web. For a low recurring cost, users get access to these apps and the connected services that tie them together in the cloud.
```

Comment:
This is a nitpick, but I wouldn't refer to their fees as "low". That's subjective.

Reply:
I could reword this, but might also link to a study. It's low in comparison to the cost of a license for Microsoft Office amortized over the average period between releases (pre-Office 365, so up until 2016). I'll grab the stats and link them.

Reply:
Done.